express_relay/sdk/js/src/examples/simpleSearcherLimo.ts

```diff
@@ -0,0 +1,217 @@
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import { Client } from "../index";
+import { BidStatusUpdate } from "../types";
+import { SVM_CONSTANTS } from "../const";
+
+import * as anchor from "@coral-xyz/anchor";
+import { Keypair, PublicKey, Connection } from "@solana/web3.js";
+
+import * as limo from "@kamino-finance/limo-sdk";
+import { Decimal } from "decimal.js";
+import {
+  getPdaAuthority,
+  OrderStateAndAddress,
+} from "@kamino-finance/limo-sdk/dist/utils";
+
+const DAY_IN_SECONDS = 60 * 60 * 24;
+
+class SimpleSearcherLimo {
+  private client: Client;
+  private connectionSvm: Connection;
+  private clientLimo: limo.LimoClient;
+  private searcher: Keypair;
+  constructor(
+    public endpointExpressRelay: string,
+    public chainId: string,
+    privateKey: string,
+    public endpointSvm: string,
+    public globalConfig: PublicKey,
+    public apiKey?: string
+  ) {
+    this.client = new Client(
+      {
+        baseUrl: endpointExpressRelay,
+        apiKey,
+      },
+      undefined,
+      () => {
+        return Promise.resolve();
+      },
+      this.bidStatusHandler.bind(this)
+    );
+    this.connectionSvm = new Connection(endpointSvm, "confirmed");
+    this.clientLimo = new limo.LimoClient(this.connectionSvm, globalConfig);
+    const secretKey = anchor.utils.bytes.bs58.decode(privateKey);
+    this.searcher = Keypair.fromSecretKey(secretKey);
+  }
+
+  async bidStatusHandler(bidStatus: BidStatusUpdate) {
+    let resultDetails = "";
+    if (bidStatus.type == "submitted" || bidStatus.type == "won") {
+      resultDetails = `, transaction ${bidStatus.result}`;
+    } else if (bidStatus.type == "lost") {
+      if (bidStatus.result) {
+        resultDetails = `, transaction ${bidStatus.result}`;
+      }
+    }
+    console.log(
+      `Bid status for bid ${bidStatus.id}: ${bidStatus.type.replaceAll(
```

Comment:
I think we dont need this part anymore. statuses are: `[pending, submitted, lost, won]`
Previously we used this line for `simulated_failed` status which is deprecated.